src/handlers/internal-server-error-handler.ts

```diff
@@ -29,12 +29,16 @@ const serverErrorHandler: ErrorRequestHandler = (err, req, res, next) => {
     return next(err);
   }
 
+  if (res.statusCode === 503) {
+      return res.render(getHtmlPath("errors", "service-unavailable-s3", ERROR_PAGES.SERVICE_UNAVAILABLE))
```

Comment:
I didn't even realise we had this 503 page..! Perhaps once we get designs etc., we can build the page statically during the build process - something we'll worry about later.